bot/cogs/help_cog.py

```diff
@@ -59,7 +59,7 @@ async def send_group_help(self, ctx, command: commands.Group):
 
         await ctx.send(embed=embed)
 
-    async def send_command_help(self, ctx, command: commands.Command):
+    async def send_command_help(self, ctx, command: ext.ClemBotCommand):
```

Comment:
What was the purpose of building a class instead of using commands.Command?

Reply:
ext.ClemBotCommand is clembots extension class to allow for the short_help long_help decorators we use

Reply:
Oh that makes sense.